tests/workbookTemplate.test.ts

```diff
@@ -0,0 +1,52 @@
+import workbookTemplate from "../src/workbookTemplate";
+import { pqUtils } from "../src/utils";
+import { pqCustomXmlPath } from "../src/constants";
+import MashupHandler from "../src/mashupDocumentParser";
+import JSZip from "jszip";
+
+describe("single query template tests", () => {
+    const singleQueryDefaultTemplate =
+        workbookTemplate.SIMPLE_QUERY_WORKBOOK_TEMPLATE;
+    let defaultZipFile;
+
+    beforeAll(async () => {
+        defaultZipFile = await JSZip.loadAsync(singleQueryDefaultTemplate, {
+            base64: true,
+        });
+    });
+
+    test("Default template is valid", async () => {
```

Comment:
this test is just for mashup - "default template is a valid zip file, and pq base64 can be extracted from it"